frigate/genai/openai.py

```diff
@@ -21,7 +21,13 @@ class OpenAIClient(GenAIClient):
 
     def _init_provider(self):
         """Initialize the client."""
-        return OpenAI(api_key=self.genai_config.api_key)
+
+        if self.genai_config.base_url is None or self.genai_config.base_url == "":
+            self.genai_config.base_url = "https://api.openai.com/v1"
```

Comment:
I don't think we should be managing the default url for OpenAI here, I'd prefer to let the upstream `openai-python` package specify the default if it is not set.

https://github.com/openai/openai-python/blob/52357cff50bee57ef442e94d78a0de38b4173fc2/src/openai/_client.py#L121